src/NzbDrone.Core.Test/InstrumentationTests/DatabaseTargetFixture.cs

```diff
@@ -68,6 +69,7 @@ public void write_long_log()
 
         [Test]
         [Explicit]
+        [ManualTest]
```

Comment:
It's dumb, but the new runner sees that we want to run the tests because they we're ignored and includes them, left `Explicit` for other runners (like resharper).